crates/driver/src/tests/setup/mod.rs

```diff
@@ -362,6 +365,13 @@ impl Solver {
     pub fn balance(self, balance: eth::U256) -> Self {
         Self { balance, ..self }
     }
+
+    pub fn rank_by_surplus_date(self, rank_by_surplus_date: chrono::DateTime<chrono::Utc>) -> Self {
```

Comment:
Given where we are now (this goes live today) I think we should use the 1-1-1970 value by default for all tests (assuming they still pass). 

Otherwise there is just more to remove during cleanup.

Reply:
Some of the non-fee related tests fail with this config enabled. Couldn't fix them quickly, so it might be better to do it during the cleanup.